pkg/scorecard_result.go

```diff
@@ -108,23 +108,24 @@ func FormatResults(
 	results *ScorecardResult,
 	doc checks.Doc,
 	policy *spol.ScorecardPolicy,
+	output io.Writer,
```

Comment:
if we're avoiding changing the signature, can we parse opts and open the file here?